src/it/aggregate-add-third-party-missing-file/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  #%L
+  License Maven Plugin
+  %%
+  Copyright (C) 2019 - Nikolas Falco
+  %%
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation, either version 3 of the
+  License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Lesser Public License for more details.
+
+  You should have received a copy of the GNU General Lesser Public
+  License along with this program.  If not, see
+  <http://www.gnu.org/licenses/lgpl-3.0.html>.
+  #L%
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.codehaus.mojo.license.test</groupId>
+  <artifactId>aggregate-add-third-party-missing-file</artifactId>
+  <version>@pom.version@</version>
+
+  <modules>
+    <module>child1</module>
+  </modules>
+
+  <name>License Test :: aggregate-add-third-party-missing-file</name>
+
+  <packaging>pom</packaging>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <license.verbose>true</license.verbose>
+    <license.failOnMissing>true</license.failOnMissing>
+    <license.useMissingFile>true</license.useMissingFile>
+    <!--
+    <license.missingFileUrl>file://${project.basedir}/THIRD-PARTY.properties</license.missingFileUrl>
+     -->
```

Comment:
Please remove the commented code or explain why it is commented.

Reply:
in my previous post I comment that the test case only cover test missingFile but **not** missingFileUrl (subject of changes) because the plugin only cover HTTP URL. I do not have idea how to provide a valid http server up and running.
So that line it's a reminder that could be de-comment if you agree with the changes on [this branch](https://github.com/nfalco79/license-maven-plugin/tree/feature/url)) where support generic URL. In that branch enable the line above to test both strategies. For 2 missing licenses, one is provided by the missingFile and the other one by missingFileUrl.

Reply:
PR #249 has that line not commented